### DISCUSSION TITLE
Kotlin

### DISCUSSION THREAD

**Discussion by thatvertigo** [mfussenegger/nvim-jdtls#115] [Q&A]
How can one configure this plugin for writing Kotlin and integrating Kotlin into java projects?

**Comment by mfussenegger** [comment]
I don't think [eclipse.jdt.ls](https://github.com/eclipse/eclipse.jdt.ls) supports kotlin. You might want to bring it up in their repository. This here is just client extensions and most of the functionality depends on what the server can do.

**Reply by thatvertigo** [reply]
Ah alright, I guess im stuck with IntelliJ for the forseeable future then :pensive:
